### ISSUE TITLE
Using quality configuration variables not registering

### ISSUE THREAD

**Issue by dawb** [wedevelopnl/silverstripe-webp-images#7]
Hi, 

I have been trying to increase the quality of images outputted but when using the example code being set in mysite.yml, it doesn't register the increase in quality. It appears to still using the value 80 which is being set in  /webmen/silverstripe-webp-images/src/WebpGenerator.php. 
The value self::$quality used at line 51 imagewebp($image, $filename, self::$quality); doesn't seem to be getting updated.

SilverStripe\Core\Injector\Injector:
  TheWebmen\WebpImages\Assets\WebpGenerator:
    properties:
      enabled: true
      quality: 90

**Comment by Dennisprins93** [comment]
Hi @dawb 

We have fixed this issue now in v0.6. You can update if you want. Let us know if you still encounter this issue afterwards. Please make sure also to put your .yml code in a seperate file, for example called 'webp.yml' and add this content:

```
---
Name: custom-webpimages
After:
  - 'webpimages'
---
SilverStripe\Core\Injector\Injector:
  TheWebmen\WebpImages\WebpGenerator:
    properties:
      enabled: true
      quality: 100
```

**Comment by dawb** [comment]
Thanks @Dennisprins93 , I have tested today and it is working correctly now.
